packs/democratic_csi_nfs/metadata.hcl

```diff
@@ -0,0 +1,11 @@
+app {
+  url    = "https://github.com/democratic-csi/democratic-csi"
+  author = "Tim Gross <tgross@hashicorp.com>"
+}
+
+pack {
+  name        = "democratic_csi_nfs"
+  description = "This pack deploys the democratic-csi plugin, configured for use with NFS"
+  url         = "https://github.com/hashicorp/nomad-pack-community-registry/democratic_csi_nfs"
```

Comment:
The docs on this `url` field are a little vague. It's not really the URL to the directory, right?

Reply:
`It's not really the URL to the directory, right?` It is... but @angrycub and I want to remove it and just make this implicit when they grab the registry itself. Because once you fork a registry, all of these are wrong.

Reply:
I more meant that our docs say to use a URL like https://github.com/hashicorp/nomad-pack-community-registry/nginx but the actual URL to the pack directory is https://github.com/hashicorp/nomad-pack-community-registry/tree/main/packs/nginx

Reply:
I've opened https://github.com/hashicorp/nomad-pack/pull/240 to update the docs and/or discuss it more generally.